kustomize/overlays/dev/secrets/event-bridge-manager-secrets.yaml

```diff
@@ -32,6 +32,7 @@ spec:
     RHOAS_OPS_ACCOUNT_CLIENT_ID: AgAZwmIcUOZJIaRIg3bx2jQhx1rU/Ej8pwK59RCTlpgtMLC0y2RO+cxmRO1lkTuemFd9hceP/9fMzrxQgFSDKmdl7gmgtQGoLvWyI9BOxrOFWlsC92CZVzpKzF/w8611g/yPlsU04vcV10vYlTGe9Ve3XllBS+UP1zqtkxRb6qd/19Nfvh+eophDPx2MpTXF5lSRm0Hrsc1hzeTOKnrGuXYOjZNP6xLmRmcABe2ImnbM6jUQk9o71RfarCsnSpAZ6NsqXvlyJTYkIpjLkUlCq0GRHxGbXY/B8xkRmDG7sgPV81QbVmMu5bwyGOAiexdyfUC678UY1QiFTwIDL0PtPhsdfir/H7Y+GiprU8Gg9vdrFsgDxGKMOcOJ6jMAHRX4oB1hRmwemVhP66fYL9tiITDthI4l6U7UggtG0/ZxtuvfCzNlS+4cfoH6JozB8tNifsQzCJn5CxTyD9jWkiOun4kmoG5002Dhq21rFn9e84pxI25W4tumzW2la9swJGLn/ZtYFP0geq2ZvP4xK2j0nGgjHvHC2BlZ3e93+UvHWfgFU7ki94n/a4EHmVXa+G8RyelZyxpfSUOHmEphFwafjqQ36MU6S35rn5jGJv3pxLnpZ4DKFFmxTiShg9ksXpkzGRxCopONWl5EaL/VRqtw5QvVPnbi/vo/uNrlyq8mDl4HLper6Ln0fUbCNOgo1zytbl9/rL14B/k2qqYXOz0FVmibpp7DPMydMFMi8d8NZIuXNt3xs99qf3/vZpeWZ/8G
     SHARD_ID: AgBk9cQqlSbuZlRbPqpq6epMdoF6uIpaihGhHuMAtujx6rDnQIIVQUO65L7UmuTrpnwrgunCRXrJAWiXRS2UJgMhSgkB1E4QyYSWtR0MwXYYqQwBefFeoP9iToL0k3PCx8CMzTunNtvj2AEAf7jPw0X77OIq8BWHYtQLB1p8xszu5dbl2VwxUIAZobL4lxspjTkuFQWhXuEIUao227IsZJ684P8eYzuSM0wMsw0LdsJts0azBuFJiZGwzN8hOxGeluPKxpa5srQwnjB7EGTnhowigMIBBiJYMULcXSYGTnRvC2GazLjjEZeuDczQfgBiYPAgj0TClTR6xqIUqCJgVw9jwWlo3zABCo2G6LqkIADaz395Qoix3aTfc+9h6VhIQg/17InYdqLdRiV1Xg+YVH8RIITYOAA1He1DHEFZADB1IMHyZmorrX05+0FHJao1QKvdq6t6oPr9dt/048Kf9ELqi1+yUBPWwIX2C5UfRmSOYnBiud2DqR0BwwaoHq+1iHo6YO4nF7ChDYLAfN6g8mOuHajB5vz0aGnnZ1eP/JCypOXU744QVbB8+2SbwALX0f8qlmXy8Bp0ko5gLRrB69GtIi1QGCGQccEWyBFgd6DQ3k/cYxWAujFceooQpeeQ3/ijoPhjOG5J1vSKGp/MrOASwbpnHxGbyWkrThxYNusYNSFM/uKE/s76pFtuYkTyRC/vYabE5y9qiQ==
+    SHARD_ROUTER_CANONICAL_HOSTNAME: AgB3vCPSCHRIfKamjNN+NGipW9lNKfHWUuRJMsuoqzDdvIX1tMoKBQ43svhQ1KBWmHFNkslxYijBHPJoVxrogn1R5m+GKpJpYYWY6Rl7Ben7Oj4mOXz+w6i2H0TNxh6ovGAPfFqyfa9DkqpxQXNaalzTFeYa63HTsAhmCCM6g3KOEMS6PsoooO0Sk328r9wzJzn8eDU1tAxTaPOnMpHgvFDVghK0AVSxf6ilxY/YQ7K6/IpxtmKNMRdNNCAel2c/hXPDLk+BduqcPZPuLfgASDmIUFTgwx+Pcwi0hQtMVGiilwOP4m/HbN8v2cWGWUnm8BKkYIH5lP4EA1/oI8JhlnVDHZmSTNe5jzeBcQHFhQvz0mZF9G3WOqPy+PgiREdkQhq3ruZWJjERMKVr3vwvGB20wk2Wc3gOmYkKY6KJDOWFw9yDrtKXkcrY5kTFai4Nov8jSM68O2C/HvJGapOnMoKH9uO0WECia2PfZGBSfn5EYrmUQXZwpE74lDQeZ1CxtgWuSvUcm5j7Sl8XA3E9aFjEBWWGpArQEYvCLmxtL9TJamaZEL3TTWI52sR58thVP6f2Qe6LQRk8kVS2SPZUPzrPtjI4e58iWo4U3JLM2OIggCSoWKy1C78IKpl+fSjpzJy3+nwavNJP3reedb5UB+Jo+lCUO4Vh0eUwcWB3AL5Red+YraMorSCXYXdP/m/R2xKo3N0EqL3NNgKrJUirtanNUsDUVZMPLZgXeWumSG5LJMaEV7o2GKt1s7WBWdzevCGXT+1BBPQ6tYDIO8s=
```

Comment:
Are these properties encoded somehow? It sounds to me like a quite strange hostname. Can't we use something simple?

Reply:
These are secrets. You can read more about how we handle secrets in `dev` and `stable` here https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/tree/main/kustomize#sealed-secrets